backend/dune

```diff
@@ -15,13 +15,13 @@
 (copy_files# "%{env:ARCH=amd64}/*.ml{,i}")
 
 (rule
+ (enabled_if (= %{env:ARCH=amd64} "arm64"))
  (targets emit.ml)
  (mode    fallback)
- (deps    amd64/emit.mlp
-          arm64/emit.mlp)
+ (deps    arm64/emit.mlp)
  (action
    (progn
      (with-stdout-to contains-input-name
-       (bash "echo %{env:ARCH=amd64}/emit.mlp"))
+       (bash "echo %{env:ARCH=arm64}/emit.mlp"))
```

Comment:
Not really important, but this line
feels a bit weird, given it is now
guarded by the condition that
`env` is `"arm64"`.

Reply:
yes, I can just remove the use of ARCH here (kept it for the case we add another target).

Reply:
My feeling is that it should either be
without `ARCH`, or with the original
fallback. But clearly I am nitpicking,
it should not be blocking.